commons-extension-webapp/src/main/webapp/ckeditor/plugins/embedbase/plugin.js

```diff
@@ -423,9 +423,12 @@
 					if ( request.task ) {
 						request.task.done();
 					}
-
-					this._setContent( request.url, evtData.html );
-					return true;
+					if ( editor.name.indexOf('comment_') > -1 && request.response.type !== 'video' ) {
```

Comment:
Use a dedicated property to distinguish from retrieving or not a preview. The name of CKEditor instances should be used to allow or not a preview. Please use the existing extension mechanism to enabled/disabled a CKEditor Feature from config and not from its name, embed the behavior in a patched CKEditor Plugin which makes it difficult to upgrade. Ckeditor Config for `activityComment` type:
https://github.com/Meeds-io/social/blob/develop/extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/ckeditor-configuration.xml#L81-L83